### PR TITLE
[EGD-8075] Add dr_libs as subrepo

### PR DESCRIPTION
**Pull request by kkleczkowski**
It adds dr_libs as third party subrepo.

### CODE REVIEW

**Review by lucckb — APPROVED** [review]

File: third-party/CMakeLists.txt [hunk 1]
```diff
@@ -23,6 +23,7 @@ add_subdirectory(sml)
 add_subdirectory(taglib)
 add_subdirectory(tinyexpr)
 add_subdirectory(utz)
+#add_subdirectory(dr_libs)
```
Comment (alekrudnik) [inline third-party/CMakeLists.txt:26]: where is the cmake for dr_libs? is it included in the subrepo?

File: .gitmodules [hunk 2]
```diff
@@ -99,3 +99,6 @@
 	path = third-party/lwext4/lwext4
 	url = ../lwext4.git
 	branch = mudita
+[submodule "third-party/dr_libs"]
+	path = third-party/dr_libs
+	url = git@github.com:mudita/dr_libs.git
```
Comment (SP2FET) [inline .gitmodules:104]: please change the URL to the HTTPS instead of SSH

Comment (kkleczkowski) [reply .gitmodules]: Fixed

File: third-party/CMakeLists.txt (hunk 1 above)
Comment (kkleczkowski) [reply third-party/CMakeLists.txt:26]: fixed

**Review by SP2FET — APPROVED** [review]

**Pull request merged**